src/corehost/common/pal.windows.cpp

```diff
@@ -250,6 +250,47 @@ bool pal::get_default_installation_dir(pal::string_t* recv)
     return true;
 }
 
+namespace
```

Comment:
(for my learning) - why the empty namespace?

Reply:
It makes the functions in it "private" to this source file. Without the namespace if there would be two source files with the same function name, the linker would complain that there are duplicates. In C# this would be a normal private static member function, but in C++ even private members must be declared in the class declaration which lives in the header file. So changes to privates still affect anybody who includes the header file - even if only causing recompilation. So the empty namespace is a way to have private functions (which don't need access to members) without the necessity to putting them to the header file.